ledger-tool/src/main.rs

```diff
@@ -2236,6 +2277,44 @@ fn main() {
                            If no file name is specified, it will print the metadata of all ledger files.")
             )
         )
+        .subcommand(
+            SubCommand::with_name("top-accounts")
+            .about("Print the top N accounts of the specified field.")
+            .arg(
+                Arg::with_name("accounts_db_path")
+                    .long("accounts-db-path")
+                    .takes_value(true)
+                    .value_name("ACCOUNTS_DB_PATH")
+                    .help("Path to the accounts_db.")
```

Comment:
I think `runtime/store-tool/` could be a better home for this and some of the other open PR's you have at the moment. This looks to only operate on accountsDB and not the ledger at all (kind of clunky to have to pass in the ledger arg if it will be unused).

Reply:
> I think store-tool only operates on a single store at a time. Probably could be expanded to do more.

Yeah, my initial thought was expanding it to do more.

> The ledger-tool unfortunately is the catch-all junk drawer for various commands

Agreed, and given that these changes are clearly accountsDB only, I think it makes sense to not throw them into the catch-all.

> Maybe these PRs could be the start of an accounts-tool?

Yeah, they could be. Kind of back to the first point, I think we should first determine if it makes sense to group these together with `store-tool`. I think operating on a single file vs the collection of file seem like similar enough use cases to tie into a single tool, but I'd probably defer to those who have been working much more with accountsDB.

One more note, if we end up adding an `accounts-tool` and it is a crate that we publish, make sure we claim the crate the first to avoid the supply chain bounty. I think CI might have been added to check for this, but doesn't hurt to mention it either.

Reply:
> Maybe these PRs could be the start of an accounts-tool?

Another thing I just noticed / remembered, `store-tool` is not currently distributed with the set of binaries included in releases. Potentially another consideration depending on whether we intend these commands to be for internal use, or if there would be scenarios where we are asking validators to run these commands on their machines while debugging

Reply:
I've been thinking about an accounts-tool as well.  I will create one and have these PRs built on top of that.